pyproject.toml

```diff
@@ -23,12 +23,21 @@ classifiers = [
 ]
 requires-python = ">=3.11"
 dependencies = [
-    "Deprecated<2",
-    "IPython<9",
-    "pyvo>=1.5.0,<2"
+    "Deprecated",
+    "IPython",
+    "pyvo",
+    "structlog",
+    "symbolicmode",
```

Comment:
Convention for our library packages is to keep the major version pins to make sure that we test before moving into uncharted territory. Also, we should keep the version constraint on pyvo, since the library won't work on earlier versions.

Reply:
OK.  Not pinning structlog, though, because:

https://github.com/hynek/structlog/blob/main/.github/SECURITY.md

```
We are following [CalVer](https://calver.org/) with generous backwards-compatibility guarantees. Therefore we only support the latest version.

Put simply, you shouldn't ever be afraid to upgrade as long as you're only using our public APIs. Whenever there is a need to break compatibility, it is announced in the changelog, and raises a DeprecationWarning for a year (if possible) before it's finally really broken.

You can't rely on the default settings and the structlog.dev module, though. They may be adjusted in the future to provide a better experience when starting to use structlog. So please make sure to always properly configure your applications.
```